x-pack/plugins/observability_solution/infra/common/snapshot_metric_i18n.ts

```diff
@@ -113,16 +113,18 @@ export const toMetricOpt = (
   nodeType?: InventoryItemType
 ): { text: string; textLC: string; value: SnapshotMetricType } | undefined => {
   switch (metric) {
-    case 'cpuTotal':
+    case 'cpuV2':
       return {
         text: Translations.CPUUsageTotal,
         textLC: TranslationsLowercase.CPUUsageTotal,
-        value: 'cpuTotal',
+        value: 'cpuV2',
       };
     case 'cpu':
       return {
-        text: Translations.CPUUsageLegacy,
-        textLC: TranslationsLowercase.CPUUsageLegacy,
+        text: showLegacyLabel(nodeType) ? Translations.CPUUsageLegacy : Translations.CPUUsageTotal,
```

Comment:
I opened an issue for this, feel free to close it https://github.com/elastic/kibana/issues/190118

Reply:
Thank you, I linked it to the PR 🙌